elasticapm/transport/http.py

```diff
@@ -106,6 +106,19 @@ def send(self, data):
                 response.close()
 
     def get_config(self, current_version=None, keys=None):
+        """
+        Gets configuration from a remote APM Server
+
+        :param current_version: version of the current configuration
+        :param keys: a JSON-serializable dict to identify this instance, e.g.
+                {
+                    "service": {
+                        "name": "foo",
+                        "environment": "bar"
+                    }
+                }
+        :return: a three-tuple of new version, config dictionary and validity in seconds, or None
```

Comment:
this is actually wrong, we *always* return a three-tuple, the elements of which can be `None`